media/img/prompt-workspace-old.svg

```diff
@@ -0,0 +1 @@
4H78.66c-3.41,0-4.47-1.67-3-4.79,2-4.37,4-8.74,6-13.12A1.6,1.6,0,0,1,82.93,225q19.45-3.41,38.91-6.86a2.73,2.73,0,0,0,.51-.07c2-.79,3.7-.06,5.46,1,5.7,3.41,11.49,6.69,17.24,10l24.67,14.23c.27.15.53.31.79.47Z"/><path class="cls-3" d="M115.83,212.22l-30.38,5.37,8.24-18.16Z"/><path class="cls-3" d="M170,193.1l-5,11.36,34.73-6.94-.06-.09a8.57,8.57,0,0,0-7.45-4.33Z"/><path class="cls-3" d="M101.77,193.1H115a7.89,7.89,0,0,1,4.62,1.5l18.77,13.65-5.78,1.24a7.84,7.84,0,0,1-5.73-1Z"/></g><g id="Layer_5" data-name="Layer 5"><circle class="cls-4" cx="131.05" cy="147.25" r="34.6"/><ellipse class="cls-5" cx="131.29" cy="147.86" rx="30.97" ry="23.96" transform="translate(-42.65 53.82) rotate(-20)"/><path class="cls-2" d="M136,160.73a15.11,15.11,0,0,0,9-19.38c-3.88-10.66-23.81-21.22-23.81-21.22s-8.48,20.91-4.6,31.56A15.1,15.1,0,0,0,136,160.73Z"/><polyline class="cls-6" points="140.52 146.47 130.26 145.07 126.46 134.65"/><path class="cls-7" d="M129,112.81a25,25,0,1,1-18.42,6.7A34.57,34.57,0,0,0,98.6,159.15c8.87,24.38,54.46,48.53,54.46,48.53s19.4-47.81,10.53-72.18A34.54,34.54,0,0,0,129,112.81Z"/></g><g id="Layer_2_copy" data-name="Layer 2 copy"><path class="cls-2" d="M295.57,130a35.08,35.08,0,1,1-35.08,35.08A35.06,35.06,0,0,1,295.57,130m70.16,20.05a20,20,0,1,1-20,20,20.06,20.06,0,0,1,20-20m-140.32,0a20,20,0,1,1-20,20,20.06,20.06,0,0,1,20-20m150.35,50.11a20.06,20.06,0,0,1,20,20.05v10a10,10,0,0,1-10,10H365.11A45.8,45.8,0,0,0,341.58,206a20,20,0,0,1,14.13-5.83h20.05m-140.33,0A20,20,0,0,1,249.56,206,46,46,0,0,0,226,240.26H205.36a10,10,0,0,1-10-10v-10a20.06,20.06,0,0,1,20-20.05h20.05m84.2,10a36.09,36.09,0,0,1,36.08,36.08v9a15,15,0,0,1-15,15H250.47a15,15,0,0,1-15-15v-9a36.09,36.09,0,0,1,36.09-36.08h2.6a48.44,48.44,0,0,0,42.91,0h2.6M295.57,122a43.13,43.13,0,0,0-43,41,28,28,0,1,0-43.47,29.9,28.09,28.09,0,0,0-21.73,27.33v10a18,18,0,0,0,18,18h22.07v7a23.07,23.07,0,0,0,23,23h90.21a23.06,23.06,0,0,0,23-23v-7h22.07a18,18,0,0,0,18-18v-10a28.08,28.08,0,0,0-21.73-27.33A28,28,0,1,0,338.6,163a43.13,43.13,0,0,0-43-41Zm21.88,80.19a43.25,43.25,0,0,0,20.42-28.9,28.11,28.11,0,0,0,11.52,19.59A27.82,27.82,0,0,0,336,200.31L332.21,204a43.94,43.94,0,0,0-12.58-1.83Zm-75.7-9.31a28,28,0,0,0,11.52-19.59,43.23,43.23,0,0,0,20.43,28.9h-2.18A44,44,0,0,0,258.93,204l-3.74-3.71a27.82,27.82,0,0,0-13.44-7.43Z"/></g><g id="Layer_2" data-name="Layer 2"><circle class="cls-5" cx="225.41" cy="170.09" r="20.05"/><circle class="cls-7" cx="365.73" cy="170.09" r="20.05"/><path class="cls-7" d="M375.76,200.16H355.71A20,20,0,0,0,341.58,206a45.8,45.8,0,0,1,23.53,34.27h20.67a10,10,0,0,0,10-10v-10A20.06,20.06,0,0,0,375.76,200.16Z"/><circle class="cls-4" cx="295.57" cy="165.08" r="35.08"/><path class="cls-4" d="M319.63,210.19H317a48.44,48.44,0,0,1-42.91,0h-2.6a36.09,36.09,0,0,0-36.09,36.08v9a15,15,0,0,0,15,15h90.21a15,15,0,0,0,15-15v-9A36.09,36.09,0,0,0,319.63,210.19Z"/><path class="cls-5" d="M249.56,206a20,20,0,0,0-14.13-5.83H215.38a20.06,20.06,0,0,0-20,20.05v10a10,10,0,0,0,10,10H226A46,46,0,0,1,249.56,206Z"/></g></svg>
```

Comment:
@zmustapha Do we need this file?

Reply:
Leave them there for a bit. I'll decide by end-of-day if I want to got back to using that image.